### ISSUE TITLE
2.2.2 download keys network error

### ISSUE THREAD

**Issue by runtman** [rancher/rancher#19702]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Cluster > Nodes > DownloadKeys

**Result:**

```
ony0/UCM9Myxy.bin.part could not be saved, because the source file could not be read.

Try again later, or contact the server administrator.
``` 

^ Firefox

```
Failed - Network Error 

0:215051 Resource interpreted as Document but transferred with MIME type application/octet-stream: x/v3/nodes/c-pfrqc:m-57qr2/nodeconfig". 
```

^ Chome

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
- Installation option (single install/HA):
HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
Infra Provider - Digital Ocean
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
Vm Droplet, 2cpu2gbram

- Kubernetes version (use `kubectl version`):

```
> kubectl version
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.7", GitCommit:"6f482974b76db3f1e0f5d24605a9d1d38fad9a2b", GitTreeState:"clean", BuildDate:"2019-03-25T02:52:13Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:19:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
18.06.1-ce
```

**Comment by runtman** [comment]
In the meantime, is there a way to get these keys manually?

**Comment by Tediferous** [comment]
I am also experiencing this 👍

**Comment by rust84** [comment]
We also experienced this on a clean install of 2.2.2.

**Comment by cjellick** [comment]
possibly related to our gzip changes

**Comment by rmweir** [comment]
@runtman @Tediferous @rust84 There is currently a fix under review for this issue. To get the keys for now you can: go to nodes, open dev tools, select networking tab, right click on request labeled 'nodeconfig', copy as curl, then paste into terminal, and remove gzip from accept-encoding field.

**Comment by rmweir** [comment]
For the actual zip file you can add -o ./save.zip in front of "curl". Replace save with your desired filename. It is still necessary to remove gzip from accept-encoding field.

**Comment by jira-sync-svc** [comment]
➤ Ricky commented:

This bug only occurs in HA mode. The cause is present in single install but only breaks HTTP2, which is used in HA.

**Comment by jira-sync-svc** [comment]
➤ Jack Luo commented:

The bug is reproduced in Rancher v2.2.2 HA mode

The big fix is validated in Rancher v2.2.3-rc1

the key can be downloaded successfully and I can ssh to the node using the key.
